src/snuffleupagus.c

```diff
@@ -71,14 +71,13 @@ PHP_GINIT_FUNCTION(snuffleupagus) {
   SP_INIT(snuffleupagus_globals->config.config_upload_validation);
   SP_INIT(snuffleupagus_globals->config.config_disabled_functions);
   SP_INIT(snuffleupagus_globals->config.config_disabled_functions_ret);
-  SP_INIT(snuffleupagus_globals->config.config_cookie_encryption);
+  SP_INIT(snuffleupagus_globals->config.config_cookie);
   SP_INIT(snuffleupagus_globals->config.config_disabled_constructs);
 
   snuffleupagus_globals->config.config_disabled_constructs->construct_include = sp_list_new();
   snuffleupagus_globals->config.config_disabled_functions->disabled_functions = sp_list_new();
   snuffleupagus_globals->config.config_disabled_functions_ret->disabled_functions = sp_list_new();
-
-  SP_INIT_HT(snuffleupagus_globals->config.config_cookie_encryption->names);
+  snuffleupagus_globals->config.config_cookie->cookies = sp_list_new();
```

Comment:
Why a `sp_list` instead of a `HashTable`? You can store whatever you want in php hashtables :)